MPSGameLift/Documentation/GameLift.md

```diff
@@ -106,14 +83,15 @@ aws gamelift create-fleet --region us-west-2 --name GameLiftO3DTest2016 --ec2-in
 **NOTE**
 
 The ec2-instance-type and fleet-type determines the kind of AWS resources used; your AWS account may incur costs.
+https://aws.amazon.com/ec2/pricing/reserved-instances/pricing/
 
 ---
 
 Record the FleetId for the next step. Example: fleet-1a49fc3e-892a-40fc-b2e9-aa7e11983182
 
 ### Create and Join Game Session
 ```sh
-aws gamelift create-game-session --region us-west-2 --fleet-id <FleetId> --name foogamesession1 --maximum-player-session-count 3
+aws gamelift create-game-session --region us-west-2 --fleet-id <FleetId> --name foogamesession1 --maximum-player-session-count 10
```

Comment:
Maybe make region something configurable or call out that we are using us-west-2 as the example region?

Reply:
@AMZN-daimini doc update fixes this.